src/sas/qtgui/Perspectives/Fitting/ConstraintWidget.py

```diff
@@ -1051,3 +1051,16 @@ def getReport(self):
         msgbox.setWindowTitle("Fit Report")
         _ = msgbox.exec_()
         return
+
+    def uncheckConstraint(self, name):
```

Comment:
This should probably also include changing the `.active` attribute of corresponding Constraint object.
Otherwise, calls like `rowHasActiveConstraint` and `rowHasActiveComplexConstraint` will behave incorrectly.

Reply:
Now the method changes the constraint attribute, so it can be used safely.